testPI2/integration/domain/transfer/ilp.test.js

```diff
@@ -24,6 +24,7 @@
  --------------
```

Comment:
should this file not be deleted if the ilp file is removed/not used anymore

Reply:
Valentin is figuring this out as part of another story and will clean it in subsequent merge.

Reply:
ok sure